_pages/integrations/newrelic.md

```diff
@@ -0,0 +1,267 @@
+---
+layout: article
+title:  "New Relic"
+permalink: integrations/newrelic/
+excerpt: "Monitor model performance metrics with New Relic"
+excerpt-short: "Monitor model performance metrics with New Relic"
+categories: [integrations]
+tags: [integrations]
+show_related: false
+image:
+    teaser: /language_logos/newrelic.svg
+---
+
+This feature is available to [Algorithmia Enterprise](/enterprise) users only.
+{: .notice-enterprise}
+
+[Algorithmia Insights](https://algorithmia.com/blog/how-to-use-algorithmia-insights-for-machine-learning-model-performance-monitoring) is a feature of Algorithmia Enterprise that provides a metrics pipeline to instrument, measure, and monitor your machine learning models by integrating with popular observability platforms such as New Relic. If you're new to Insights, you can check out our [documentation](https://algorithmia.com/developers/algorithmia-enterprise/algorithmia-insights/) and learn how to configure it for your organization.
```

Comment:
"configure it for your org" --> "configure this feature for your org"

otherwise "it" sounds like it refers to "documentation"